contracts/contracts/buyback/OETHBuyback.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { BaseBuyback } from "./BaseBuyback.sol";
+
+contract OETHBuyback is BaseBuyback {
+    // abi.encodePacked(oeth, uint24(500), weth9, uint24(3000), ogv);
+    bytes public constant ogvPath =
+    // solhint-disable-next-line max-line-length
+        hex"856c4efb76c1d1ae02e20ceb03a2a6a08b0b8dc30001f4c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000bb89c354503c38481a7a7a51629142963f98ecc12d0";
```

Comment:
I think the `OETHBuyback` and `OUSDBuyback` child contracts only differ in the path strings. Maybe those could be the only override differences between them?

Reply:
I thought of that. But you cannot override constant variables in Solidity. The other way is to define `ogvPath` and `cvxPath` as virtual functions on `BaseBuyback` and then override them with public constant in `OETHBuyback` and `OUSDBuyback` and override it as a function on `MockBuyback` (for unit tests). I can give it a try though

Reply:
you could have a mapping of address => bytes where ogv address would map to => bytes path. And have that mapping populated as part of specific buyback constructor body implementation. 

Though having bytes as value might be complicated with mapping and not an elegant solution. 

So if above can't be cleanly implemented your current solution is probably the cleanest

Reply:
Having the internal method on each final contract that loads these in seems pretty reasonable. The rest of the code cleans up and deduplicates nicely.